src/index.html

```diff
@@ -1,13 +1,457 @@
+    <link rel="stylesheet" href="./styles/style.scss">
     <script src="scripts/main.js" defer></script>
   </head>
-  <body>
-    <h1>Landing Page</h1>
+  <body class="page__body">
+    <header class="header">
+      <div class="container">
+        <div class="header__content">
+          <div class="top-bar">
+            <a
+              href="#"
+              alt="THE MET logo"
+              class="top-bar__logo"
+            >
+            </a>
+            <div class="top-bar__icons">
+              <a href="tel:12125357710" class="icon icon--phone">
+                <div class="icon__numbers">
+                  <p class="icon__numbers__phone--title">
+                    The Met Fifth Avenue
+                  </p>
+                  <p class="icon__numbers__phone">
+                    +1 212-535-7710
+                  </p>
+                  <p class="icon__numbers__phone--title">
+                    The Met Cloisters
+                  </p>
+                  <p class="icon__numbers__phone">
+                    +1 212-923-3700
+                  </p>
+                </div>
+              </a>
+
+              <a href="#menu" class="icon icon--menu"></a>
+            </div>
+          </div>
+          <h1 class="header__title">
+            Welcome<br>
+            to The MET
+          </h1>
+        </div>
+      </div>
+    </header>
+
+    <aside class="page__menu menu" id="menu">
+      <div class="container">
+        <div class="top-bar">
+          <a
+            href="#"
+            alt="THE MET logo"
+            class="top-bar__logo"
+          >
+          </a>
+          <a href="#" class="icon icon--close"></a>
+        </div>
+        <div class="menu__bottom">
+          <nav class="nav menu__nav">
+            <ul class="nav__list">
+              <li class="nav__item">
+                <a href="#" class="nav__link">
+                  Home
+                </a>
+              </li>
+              <li class="nav__item">
+                <a href="#about-us" class="nav__link">
+                  About Us
+                </a>
+              </li>
+              <li class="nav__item">
+                <a href="#now-on-view" class="nav__link">
+                  Now on view
+                </a>
+              </li>
+              <li class="nav__item">
+                <a href="#online-explore" class="nav__link">
+                  Explore Online
+                </a>
+              </li>
+              <li class="nav__item">
+                <a href="#contacts" class="nav__link">
+                  Contacts
+                </a>
+              </li>
+            </ul>
+          </nav>
+          <div class="numbers">
+            <a href="tel:12125357710" class="numbers__phone">
+              +1 212-535-7710
+            </a>
+            <p class="numbers__phone--title">
+              The Met Fifth Avenue
+            </p>
+            <a href="tel:12129233700" class="numbers__phone">
```

Comment:
Need to add it everywhere
```suggestion
            <a href="tel:+12129233700" class="numbers__phone">
```